x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -2,6 +2,50 @@
 
 The `security_solution/cypress` directory contains functional UI tests that execute using [Cypress](https://www.cypress.io/).
 
+Currently with Cypress you can develop `functional` tests and coming soon `CCS` and `Upgrade` functional tests.
+
+If you are still having doubts, questions or queries, please feel free to ping our Cypress champions:
+
+- Functional Tests:
+  - Gloria Hornero (@Glo) and Patryk Kopycinsky (@patryk.kopycinski) 
+  
+- CCS Tests:
+  - Technical questions around the https://github.com/elastic/integration-test repo:
+     - Domenico Andreoli (@dom)
+  - Doubts regarding testing CCS and Cypress best practices:
+     - Gloria Hornero (@Glo)       
+
+## Table of Contents
+
+[**How to add a new Cypress test**](#how-to-add-a-new-cypress-test)
+
+[**Running the tests**](#running-the-tests)
+
+[**Debugging your test**](#debugging-your-test)
+
+[**Folder structure**](#folder-structure)
+
+[**Test data**](#test-data)
+
+[**Development Best Practices**](#development-best-practices)
+
+[**Test Artifacts**](#test-artifacts)
+
+[**Linting**](#linting)
+
+## How to add a new Cypress test
+
+Before adding a new test please ask yourself the following question: `Can I test this behaviour with the unit or API tests?` if the answer is `Yes I can`, 
+then, please add the test there since Cypress is not the best place for it :) 
+
+If the answer is `No, I can't` then this is the best place to start.
+
+First of all please take a look to the [**Development Best Practices**](#development-best-practices) seection.
+Then check check [**Folder structure**](#folder-structure) section to know where is the best place to put your test, [**Test data**](#test-data) section if you need to create any type
+of data for your test, [**Running the tests**](#running-the-tests) to know how to execute the tests and [**Debugging your test**](#debugging-your-test) to debug your test if needed.
+
+Please, before open a PR with the new test, please make sure that the test fail. If you never see your test fail you don’t know if your test is actually testing the right thing, or testing anything at all.
```

Comment:
nit: `before opening` instead of `before open`, `test fails` instead of `test fail`.

++ 👍  to this line with "red light", "green light" type of things.